Gruntfile.js

```diff
@@ -89,6 +89,7 @@ module.exports = function(grunt) {
 		connect: {
 			server: {
 				options: {
+					hostname: '0.0.0.0',
```

Comment:
Was there a reason why you needed to set this explicitly? According to [this ticket](https://github.com/gruntjs/grunt-contrib-connect/issues/60), 0.0.0.0 is already the default hostname since v0.7. I'd rather not set it if we don't need to.